packages/neutrino-preset-web/index.js

```diff
@@ -99,6 +99,10 @@ module.exports = (neutrino) => {
       .chunkFilename('[id].[chunkhash].js')
       .end()
     .resolve
+    .alias
+        // TIP: make sure 2 version of "core-js" always match in package.json and babel-polyfill/package.json
```

Comment:
Yes. I thought what is the best here. "Make sure 2 versions of "core-js" always match in package.json and babel-polyfill/package.json" sounds like the next line of code do this. But it actually not. This comment should play a role of a reminder.